classes/class-wc-connect-shipping-label.php

```diff
@@ -353,19 +353,26 @@ protected function get_form_data( WC_Order $order ) {
 
 		protected function get_states_map() {
 			$result = array();
-			foreach ( WC()->countries->get_countries() as $code => $name ) {
-				$result[ $code ] = array( 'name' => html_entity_decode( $name ) );
-			}
-			foreach ( WC()->countries->get_states() as $country => $states ) {
-				$result[ $country ]['states'] = array();
-				foreach ( $states as $code => $name ) {
-					if ( 'US' === $country && in_array( $code, $this->unsupported_states ) ) {
-						continue;
+			$all_countries = WC()->countries->get_countries();
+			$allowed_countries = array( 'US', 'PR' );
```

Comment:
What do you think about moving this to a class property so it can be reused in `calculate_should_show_meta_box()`?